pf/internal/plugin/provider_server.go

```diff
@@ -44,9 +43,11 @@ func NewProviderServerWithContext(provider ProviderWithContext) pulumirpc.Resour
 
 func (p *providerServer) unmarshalOptions(label string) pl.MarshalOptions {
 	return pl.MarshalOptions{
-		Label:         label,
-		KeepUnknowns:  true,
-		KeepSecrets:   true,
+		Label:        label,
+		KeepUnknowns: true,
+		// TF doesn't handle secrets. Since we can't do anything with secrets, we
```

Comment:
I think this comment is misleading. PF bridge is still responsible to populate secrets in its outputs.

This is unmarshalOptions though. Is this used consistently?

We can try RejectSecrets: true here explaining that since we return this further down:

```
&pulumirpc.ConfigureResponse{AcceptSecrets: false}
```

Then we do not expect the engine to send any secrets as inputs. And doing so is an error.

Reply:
We don't actually have a `RejectSecrets` option. I'll fix the comment.